docs/development/secrets_management.md

```diff
@@ -69,6 +71,20 @@ if err != nil {
 
 As explained above, this returns the bundle secret for the CA `my-ca` which might potentially contain both the current and the old CA (in case of rotation/regeneration).
 
+## Reusing the SecretsManager in Other Components
```

Comment:
Can you somehow link to this section from `docs/extensions`? I haven't checked which place might be appropriate, though.